packages/gatsby/cache-dir/develop-static-entry.js

```diff
@@ -27,7 +27,9 @@ try {
 Html = Html && Html.__esModule ? Html.default : Html
 
 export default (pagePath, callback) => {
-  let headComponents = []
+  let headComponents = [
+    <meta key="env:development" name="note" content="env:development" />,
```

Comment:
Can we stick to `<meta name="note" content="environment=development">` as proposed in the issue?

Reply:
Oh! My bad 😅 Sure sure, let me update it and let you know ✍️